flexx/util/getresource.py

```diff
@@ -80,7 +83,8 @@ def _fetch_file(url):
     info('Downloading %s' % url)
     for tries in range(4):
         try:
-            return urlopen(url, timeout=5.0).read()
+            req = Request(url, headers={'User-Agent': 'flexx'})
+            return urlopen(req, timeout=5.0).read()
```

Comment:
For some reason https://unpkg.com/ (source of online leaflet dist) requires a user agent to be set, or it will return a 403 forbidden. Having it set, shouldn't otherwise do any harm.